### PR TITLE
webapp

### PR DESCRIPTION
**Pull request by emiliazm**
Hello!

### These are the features that we implemented:

- The home page.
  - When the page loads, the webapp retrieves data from:
  - The selected API and shows the list of items on screen.
  - The Involvement API to show the item likes.
  - Remember that your page should make only 2 requests:
  - One to the base API.
  - And one to the Involvement API.
  - When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
  - When the user clicks on the "Comments" button, the Comments popup appears.
  - When the user clicks on the "Reservations" button, the Reservations popup appears (only for the groups of 3 students).
  - Home page header and navigation similar to the given mockup.
  - Home page footer similar to the given mockup.
- The comments popup.
  - When the popup loads, the webapp retrieves data from:
  - The selected API and shows details about the selected item.
  - The Involvement API to show the item comments.
  - When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.
- We have counters in all the interfaces that show:
  - The number of items (home).
  - The number of comments (comments popup).
 
### These is the link of our video
https://drive.google.com/file/d/1CaLf10gCDWhJJ-r72CZiFrbRV9kxodfr/view?usp=sharing

### Live Demo
https://emiliazm.github.io/webapp/

### CODE REVIEW

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @emiliazm  & @aliveGUY,

Well done! Well done! Well done! 👏🏿. It's evident you gave it your all 👍🏿.
Your capstone project is nicely implemented and the expected features are functioning as expected 💯.

### Some highlights for you
* Correct gitflow practice ✔️ 
* Good commit messages ✔️ 
* Professional README ✔️ 
* Nice video presentation ✔️ 
* In the project:
   * Code is ES6 compliant ✔️ 
   * Clean codebase ✔️ 
   * API interaction properly implemented using `async/await`✔️  
   * All expected features of the MVP functioning as expected ✔️

We, however, have an issue to take care of before we could move ahead. Continue reading.

## Required Changes ♻️
- [ ] I believe you've submitted this [VIDEO](https://drive.google.com/file/d/1xPl3Y2A0kEnSh75ry4i3L9mbi0CtrU89/view) by mistake. In this video, a presentation of an Elevator Pitch is all that can be realized and not the project presentation.
  We still need to fulfill these requirements 👇🏼:
  
  > - Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:
  >    - A description of the project.
  >    - A demo of the project features.
  >    - You should also highlight some interesting piece of code or something you built that you are very proud of.
  >    - You all should appear in the video and talk about the project.
  
  Please update the video submission.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by emiliazm** [comment]
Hello @Lameck1 !
Thank you for your comments!

### Changes that we implemented:
New video link: https://drive.google.com/file/d/1CaLf10gCDWhJJ-r72CZiFrbRV9kxodfr/view?usp=sharing

File: README.md [hunk 1]
```diff
@@ -1,72 +1,100 @@
...
 
-To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
+## Live Demo (if available)
 
-### Install
+[Live Demo Link] (https://emiliazm.github.io/webapp/)
```
Comment (Lucas-Masaba) [inline README.md]: [OPTIONAL]
- I think the closing square bracket and opening bracket should be placed close to each other so the link could look better. Currently, it looks like this 
image

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @emiliazm and @aliveGUY  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Awesome implementing the changes requested by the previous reviewer 🏄 
- Linter checks passing ✔️ 
- Nice animations added to the web app 🏂 

Congratulations 🥳 . Your captsone project is complete 🏂 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**